src/components/Pagination.tsx

```diff
@@ -154,64 +167,110 @@ export function Pagination<Entry>({
     updateEntriesForPageNumber(number);
   }
 
-  function nextPage() {
+  function goToNextPage() {
```

Comment:
I also updated a couple of these function names just to be super clear about what they do